app/src/main/java/com/readrops/app/more/preferences/PreferencesScreen.kt

```diff
@@ -210,6 +218,36 @@ class PreferencesScreen : AndroidScreen() {
                                 title = stringResource(id = R.string.open_items_in),
                                 onValueChange = {}
                             )
+
+                            var showDialog by remember { mutableStateOf(false) }
```

Comment:
Could you put this in `PreferencesScreenModel`? Any kind of state should come from it

Reply:
This is purely UI state. This doesn't need to be remembered accross the whole app. Are you sure it needs to go in a ViewModel?